### PR TITLE
Schreefstock fixes

### PR DESCRIPTION
**Pull request by noideas4thisname**
Fixes:
- The ability to set up accounts or respond to bots
- The "feature" to steal money with a negative transaction
- Channel spam by implementing DMs (and some handling to prevent crashes in DM channels)
 + the ability to note "here" and still post to channel if desired
- Some small changes to make Azure Table Storage store items as Numbers
- Adds nodemon for local testing

....oh and I reset all your accounts already.

### CODE REVIEW

File: handlers/stock.js [hunk 1]
```diff
@@ -63,51 +68,69 @@ function stock(channel, message, params) {
...
-                        else {
-                            channel.send('Not enough stock to make the transaction.')
-                        }
-                    });
-                }
-                else {
-                    channel.send('That user does not exist! Schreef won\'t throw his stock to the abyss.');
-                }
-            });
+        if (sendAmount < 0) {
```
Comment (thekanbo) [inline handlers/stock.js]: Should also add case for `sendAmount === 0`

File: handlers/stock.js [hunk 2]
```diff
@@ -63,51 +68,69 @@ function stock(channel, message, params) {
...
+                                LOG.info(transactionLog);
+                                channel.send(transactionString);
+                            }
+                            else {
+                                channel.send('Not enough $$ to make the transaction. Get a job.')
+                            }
+                        });
+                    }
+                    else {
+                        channel.send('Nobody of that name exists or they\'re a bot. Schreef won\'t throw his $$ to the abyss or filthy machines.');
```
Comment (thekanbo) [inline handlers/stock.js]: Maybe change Schreef explicit reference to use client user name

**Comment by skeltonath** [comment]
You're definitely gonna wanna change some things after my trigger PR is merged.

Also you should lint this as well.

**Comment by noideas4thisname** [comment]
All lint changes included. Merging now and I'll use a new PR after trigger changes.

**Pull request merged**